source/sonata_simulation.rst

```diff
@@ -452,6 +452,33 @@ Note: fields marked Mandatory* depend on which ornstein_uhlenbeck version is sel
    represents_physical_electrode  boolean    Optional     Default is False. If True, the signal will be implemented using a NEURON SEClamp mechanism, if a conductance source, or a NEURON IClamp mechanism, if a current source. The SEClamp and IClamp produce an electrode current which is not included in the calculation of extracellular signals, so this option should be used to represent a physical electrode. If the noise signal represents synaptic input, `represents_physical_electrode` should be set to False, in which case the signal will be implemented using a ConductanceSource mechanism or a MembraneCurrentSource mechanism, which are identical to SEClamp and IClamp, respectively, but produce a membrane current, which is included in the calculation of the extracellular signal.
    ============================== ========== ============ ==========================================
 
+spatially_uniform_e_field (extracellular_stimulation)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Generates an temporally-oscillating extracellular potential field.
+The potential field is defined as the sum of an arbitrary number of potential fields which vary cosinusoidally in time, and whose gradient (i.e., E field) is constant.
+
+.. table::
+
+   ============================ ========== =========== ==========================================
+   Property                     Type       Requirement Description
+   ============================ ========== =========== ==========================================
+   fields                       list       Mandatory   A list of dicts, where each dict defines one of the fields which are summed to produce the total stimulus. The format for each such dict is given in the table below.
+   ramp_up_time                 float      Optional    Duration during which the signal ramps up linearly from 0, in ms. If not provided, assume no ramp-up time (note that the specified "duration" parameter is not inclusive of this ramp-up time)
+   ramp_down_time               float      Optional    Duration during which the signal ramps down linearly to 0, in ms. If not provided, assume no ramp-down time (note that the specified "duration" parameter is not inclusive of the ramp-down time)
```

Comment:
nitpick: I find these 2 definitions a little too generic. Ramping up and down what? Amplitude? Frequency? Phase? 
```suggestion
   ramp_up_time                 float      Optional    Duration during which the amplitude of the signal ramps up linearly from 0, in ms. If not provided, assume no ramp-up time (note that the specified "duration" parameter is not inclusive of this ramp-up time)
   ramp_down_time               float      Optional    Duration during which the amplitude of the signal ramps down linearly to 0, in ms. If not provided, assume no ramp-down time (note that the specified "duration" parameter is not inclusive of the ramp-down time)
```